CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Changed
+
+- You can now pass arguments to `cairo-profiler` or `cairo-coverage`,
+  e.g. `snforge test --coverage --include tests-functions`. If flags name duplicates a flag in `snforge test`, you can
+  use `--` to separate them, e.g. `snforge test --coverage -- --help`
+- You can't use now `--coverage` and `--build-profile` flags at the same time. If you want to use both, you need to run
```

Comment:
We can change this behavior to: 
> you can't use the `--coverage` and `--build-profile` flags if you are passing additional arguments

but it might  confuse users. won't be having breaking change though